File: GestionFormationsBackend/src/main/java/Dto/Candidature/CandidatureResponse.java
package Dto.Candidature;

import Enum.StatutInscription;
import lombok.*;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CandidatureResponse {

    private Long id;
    private String nom;
    private String prenom;
    private String email;
    private Integer anneesExperience;
    private String specialites;
    private StatutInscription status;

    private List<String> files; // chargé seulement dans /files
}

File: Gestion-formation---front/src/app/core/dto/Candidature/CandidatureResponse.ts
import { StatutInscription } from "src/app/shared/utils/Enum/StatusInscription";

export interface CandidatureResponse {

  id: number;
  nom: string;
  prenom: string;
  email: string;
  anneesExperience: number;
  specialites: string;
  status: string;

  files?: string[];

}